rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/LicenseSpecTest.java

```diff
@@ -146,6 +146,8 @@ public void should_document_get_license() throws Exception {
                                 fieldWithPath("additionalData").description("A place to store additional data used by external tools"),
                                 fieldWithPath("text").description("The license's original text"),
                                 fieldWithPath("checked").description("The information, whether the license is already checked, optional and defaults to true"),
+                                fieldWithPath("OSIApproved").description("The OSI aprroved information"),
+                                fieldWithPath("FSFLibre").description("The FSF libre information"),
```

Comment:
Please change the description to add the possible values as shown below.:
`fieldWithPath("OSIApproved").description("The OSI aprroval information, possible values are: " + Arrays.asList(Ternary.values())),`

`fieldWithPath("FSFLibre").description("The FSF libre information, possible values are: " + Arrays.asList(Ternary.values())),`